eng/scripts/Initialize-VcpkgRelease.ps1

```diff
@@ -38,29 +31,24 @@ param (
 # If there's nothing in the "port" folder to upload set SkipVcpkgUpdate to true
 # and exit. Other steps will check SkipVcpkgUpdate to decide whether to move
 # forward.
-if (!(Get-ChildItem -Path "$SourceDirectory/port")) {
+if (!(Get-ChildItem -Path "$SourceDirectory/port/CONTROL")) {
     Write-Host "###vso[task.setvariable variable=SkipVcpkgUpdate]true"
     exit
 }
 
 $packageSpec = Get-Content -Raw -Path $PackageSpecPath | ConvertFrom-Json
-$assetFilename = "$($packageSpec.packageName).tar.gz"
+$tarGzUri = "https://github.com/$GitHubRepo/archive/$($packageSpec.packageName).tar.gz" 
+
+Write-Host "Downloading tarball to compute hash" 
+$localTarGzPath = New-TemporaryFile
+Invoke-WebRequest -Uri $tarGzUri -OutFile $localTarGzPath
 
-# Upload the asset to the release
-Write-Verbose "Uploading asset: $assetFilename..."
-$assetInfo = & $PSScriptRoot/../common/scripts/New-ReleaseAsset.ps1 `
-    -ReleaseTag $packageSpec.packageName `
-    -AssetPath $SourceDirectory/$assetFilename `
-    -GitHubRepo $GitHubRepo `
-    -GitHubPat $GitHubPat
+$sha512 = (Get-FileHash -Path $localTarGzPath -Algorithm SHA512).Hash.ToLower()
+Write-Host "SHA512: $sha512"
 
-$sha512 = (Get-FileHash -Path "$SourceDirectory/$assetFilename" -Algorithm SHA512).Hash
+Write-Verbose "Writing the SHA512 hash"
+$portfileLocation = "$SourceDirectory/port/portfile.cmake"
```

Comment:
nit:
```suggestion
$portFileLocation = "$SourceDirectory/port/portfile.cmake"
```

Reply:
This is named `portfile` because the file itself is called `portfile`: 

![image](https://user-images.githubusercontent.com/2158838/104407953-b25d4d80-5517-11eb-8e3a-44dc4dfd16d4.png)